packages/devtools_app/lib/src/charts/chart_controller.dart

```diff
@@ -263,7 +281,15 @@ class ChartController extends DisposableController
 
   double get fixedMaxY => _fixedMaxY;
 
-  String title;
+  String get title => _title;
+
+  set title(String newTitle) {
```

Comment:
nit: instead of calling this variable newTitle, maybe call it `value`

Reply:
done.